Assets/Tests/KotlinTokenizer/overrideargs.swift

```diff
@@ -0,0 +1,5 @@
+class Test {
+    override func dostuff(x:Int = 5) {
+    }
```

Comment:
great! this is the kind of tests useful. I am going to add a second method with default arguments not overriden to check they are not removed in that case